shlr/ptrace-wrap/include/ptrace_wrap.h

```diff
@@ -22,6 +22,12 @@
 #include <semaphore.h>
 #include <sys/ptrace.h>
 
+#ifndef __GLIBC__
```

Comment:
i prefer positive conditionals #ifdef ...

Reply:
changed it